### ISSUE TITLE
(babel plugin) TypeError: path.isClassPrivateMethod is not a function

### ISSUE THREAD

**Issue by FezVrasta** [developit/microbundle#258]
As soon I run `microbundle build` I get this error.

```
TypeError: path.isClassPrivateMethod is not a function
    at verifyUsedFeatures (node_modules/microbundle/node_modules/@babel/helper-create-class-features-plugin/lib/features.js:46:12)
```

I'm running it from a sub-folder of a monorepo, I'm not sure if that may be related. The monorepo root uses react-styleguidist and create-react-app 2.0

**Comment by developit** [comment]
I believe this is a babel versioning issue.  I was able to resolve it when I encountered it locally by wiping node_modules and reinstalling.

**Comment by FezVrasta** [comment]
Thanks for the reply, I deleted all the node_modules and reinstalled everything but I still get the same error.

Here's the output of npm ls:
```
❯ npm ls @babel/plugin-proposal-class-properties
emotion-button@1.0.0 ui/packages/emotion-button
└── (empty)
```

**Comment by ForsakenHarmony** [comment]
I don't mean microbunle, the babel plugin might be

**Comment by ForsakenHarmony** [comment]
also did you run `npm ls` after installing it? because it won't output anything if you delete `node_modules` first

**Comment by FezVrasta** [comment]
Yes I ran it after installing.

`@babel/plugin-proposal-class-properties` (according to yarn.lock) is required only by babel-preset-react-app@6.1.0 and microbundle@0.8.3, both of them require the version 7.1.0

**Comment by paulmelnikow** [comment]
This error also occurs in a non-`microbundle` project when updating `@babel/plugin-proposal-class-properties` to 7.2.0: badges/shields#2449.

**Comment by paulmelnikow** [comment]
And… looks like it's been fixed in 7.2.1: babel/babel#9120 badges/shields#2455

**Comment by developit** [comment]
Fixed in Babel.
